editoast/src/models/projects.rs

```diff
@@ -65,6 +65,35 @@ pub struct ProjectWithStudies {
     pub studies: Vec<i64>,
 }
 
+#[derive(Debug, Clone, Deserialize)]
+pub enum OrderingProject {
+    NameAsc,
+    NameDesc,
+    CreationDateAsc,
+    CreationDateDesc,
+    LastModifiedDesc,
+    LastModifiedAsc,
+}
+
+impl OrderingProject {
+    pub fn default() -> Self {
+        OrderingProject::LastModifiedDesc
+    }
+}
```

Comment:
You should implement the `Default` trait and use the std Default macro for that.

```suggestion
#[derive(Debug, Clone, Deserialize, Default)]
pub enum OrderingProject {
    NameAsc,
    NameDesc,
    CreationDateAsc,
    CreationDateDesc,
    #[default]
    LastModifiedDesc,
    LastModifiedAsc,
}
```